### PR TITLE
login.html when navigating to subpages, ui fixes for pagination on works/grants, browse by updates, Fixes #169 #220

### LINKED ISSUE #169: AE - Person page design
Create the researcher / person page based on Figma designs, like-
Screenshot 2023-09-20 at 12 02 48

The api currently exists at `/api/person/<personId>` to populate this page.

### PR DESCRIPTION
**Pull request by UcDust**

### CODE REVIEW

**Comment by UcDust** [comment]
@qjhart @Vensberg 

This addresses the following items:

-fix for browse AZ, can still click inactive letters
-search loading, … for number of results instead of showing the number of results from the previous search
-fix login, not requiring login if nav to places other than home, like http://localhost/search/x directly
-grants pagination. If I click on page 2 of grants, then click on "return to profile" then click back on grants, page 2 opens up. This happens even if I navigate to multiple pages in between returning to "see all grants". 
-same for works list

File: ae_new_project
Comment (qjhart) [inline ae_new_project:1]: What's with this new sub-module.  This looks wrong.

**Review by qjhart — CHANGES_REQUESTED** [review]
Plz remove the submodule.

File: ae_new_project
Comment (UcDust) [reply ae_new_project:1]: Good call, that should be removed now.

**Review by qjhart — APPROVED** [review]

**Pull request merged**